job_controller/job_controller.py

```diff
@@ -0,0 +1,80 @@
+"""
+Main class, creates jobs by calling to the jobcreator, creates the jobwatcher for each created job, and recieves
+requests from the topicconsumer.
+"""
+import os
+import threading
+import uuid
+from pathlib import Path
+from typing import Dict
+
+from job_controller.job_watcher import JobWatcher
+from job_controller.job_creator import JobCreator
+from job_controller.script_aquisition import acquire_script
+from job_controller.topic_consumer import TopicConsumer
+from job_controller.utils import create_ceph_path, logger
+
+
+class JobController:
+    """
+    This is the JobController class that will communicate between the consumer and the kubernetes API, it effectively
+    functions as a main class.
+    """
+
+    def __init__(self) -> None:
+        self.ir_api_ip = "irapi.ir.svc.cluster.local"
+        self.kafka_ip = os.environ.get("KAFKA_IP", "broker")
```

Comment:
```suggestion
        self.kafka_ip = os.environ.get("KAFKA_IP")
```

IMO this default would never be valid so may as well not be a default.

Reply:
It makes sense in the scenario this code was originally copied from (run-detection's implementation). Right now there is no point, as the end to end testing is not being added in this PR.